auto_nag/scripts/survey_sec_bugs.py

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+
+
+class SurveySecurityBugs(BzCleaner):
+    LIST_OF_PEOPLE_TO_REACH_OUT = [
```

Comment:
nit: put that stuff in tools.json file
```json
"survey_sec_bugs": { "to_reach_out": ["foo", "bar", ...]}
```
and then just use `self.get_config("to_reach_out", default=[])` when needed in the python code